CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [9.4.1] - 2020.10.05
+### Fixed
+  - Plugin may not validate files on Windows OS and report SUCCESS [#399](https://github.com/JLLeitschuh/ktlint-gradle/issues/399)
```

Comment:
Shouldn't this be in the present tense, instead of the past tense. IE:
"Plugin now correctly validates files on Windows OS and reports SUCCESS"

Reply:
Force-pushed changelog update